api/src/db/models/definitions/fields.ts

```diff
@@ -31,6 +33,7 @@ export interface IFieldGroup {
   description?: string;
   lastUpdatedUserId?: string;
   isVisible?: boolean;
+  isVisibleInDetail?: boolean;
```

Comment:
declare an interface with visibility related fields and use it as extension below